### ISSUE TITLE
Add Splitting Delegations

### ISSUE THREAD

**Issue by Callum-A** [eco-stake/restake#5]
Hi all,

Really like the work you've done here after a skim read over myself.

I think a cool feature (I was thinking of building a dApp for this myself but you guys have beaten me to it :)) would be to split delegations equally across a number or all restake validators. Say I want to delegate 1000 JUNO, and there are 5 restake validators, they would each receive 200 JUNO. I think it would help promote decentralisation across the network.

Especially if more validators onboard with this.

Apologies if this is in the wrong place, but happy to hear your thoughts.

**Comment by tombeynon** [comment]
Hey @Callum-A - love this idea! Right now it would only be the manual claim/compounding this would affect as the validator is only authorised to delegate to themselves (although have some ideas here in the future too).

I think what would be really nice.. is a 'bulk actions' feature. Checkboxes to choose the validators you want to affect, then options to:
- Delegate to multiple (splitting the total)
- Claim from multiple (currently can do all or 1)
- Compound multiple (takes the total rewards from selection and splits between selection)

Potentially the boring stuff like undelegate/redelegate too.

Would fit in nicely into the UI and not a challenging thing to build. I'll get this into the todo list asap!

**Comment by Callum-A** [comment]
> Hey @Callum-A - love this idea! Right now it would only be the manual claim/compounding this would affect as the validator is only authorised to delegate to themselves (although have some ideas here in the future too).
> 
> I think what would be really nice.. is a 'bulk actions' feature. Checkboxes to choose the validators you want to affect, then options to:
> 
> * Delegate to multiple (splitting the total)
> * Claim from multiple (currently can do all or 1)
> * Compound multiple (takes the total rewards from selection and splits between selection)
> 
> Potentially the boring stuff like undelegate/redelegate too.
> 
> Would fit in nicely into the UI and not a challenging thing to build. I'll get this into the todo list asap!

Yeah I definitely support the addition of mass actions. Nice work!
